mozconfig.defaults

```diff
@@ -0,0 +1,13 @@
+ac_add_options --enable-project=js
+ac_add_options --enable-application=js
+ac_add_options --target=wasm32-unknown-wasi
+ac_add_options --without-system-zlib
+ac_add_options --without-intl-api
+ac_add_options --disable-jit
+ac_add_options --disable-shared-js
+ac_add_options --disable-shared-memory
+ac_add_options --disable-tests
+ac_add_options --disable-clang-plugin
+ac_add_options --enable-jitspew
+ac_add_options --enable-optimize
+ac_add_options --enable-js-streams
```

Comment:
It looks like this option has been removed in the updated gecko-revision
https://github.com/fastly/spidermonkey-wasi-embedding/actions/runs/5060778463/jobs/9084101723?pr=15#step:4:284

Reply:
If we update the revision to 
0c6ce134d5358dc92ab5959a2b65d6dd5ba9506a that should resolve this issue